### ISSUE TITLE
Feature request: Show GPU Model in the left bar

### ISSUE THREAD

**Issue by pktiuk** [nokyan/resources#13]
I think it would be much better to have better description than GPU 1, GPU 2
![image](https://github.com/nokyan/resources/assets/45544416/17de92d2-6979-43f0-aff7-001b4d9ddc53)

It would be good to have name like `GPU GTX 1080`, `GPU Intel`, etc.

**Comment by nokyan** [comment]
Thank you, I'm glad you like it. :)
Yeah, seems like a good idea. Though I'd prefer the name of the tabs in the sidebar to be a bit more "accessible". I've thought maybe something like "Integrated GPU" and "Dedicated GPU"? Most people these days don't have more than one dedicated GPU anyways.

**Comment by pktiuk** [comment]
>  I've thought maybe something like "Integrated GPU" and "Dedicated GPU"

This is also a good idea, but during implementation remember about cases, when a device has more than one dedicated GPU (multiGPU desktops, laptops with eGPU etc.)

**Comment by nokyan** [comment]
That's true as well. I'd also need to figure out how to find out whether a GPU is integrated or not, hopefully without a self-made lookup table

**Comment by pktiuk** [comment]
For dealing with some of the GPU-related issues you could look at nvtop implementation:  https://github.com/Syllo/nvtop

Maybe one day there would be a universal library for handling GPU in Linux https://github.com/Syllo/nvtop/issues/206
